epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -53,69 +55,82 @@
-				trade sector.</p>
+			<p>EPUB is the standard format for the creation and distribution of publications in the digital publishing sector.</p>
 
 			<p>The specifications of the EPUB standard are based on the <a
-					href="https://www.w3.org/wiki/Open_Web_Platform">Open Web Platform</a>: HTML, CSS, JavaScript and
-				SVG are the core technologies used for content creation. The use of the latter means that EPUB
-				publications can be written with a high degree of accessibility simply through the proper application of
-				established web accessibility techniques, namely the <a
-					href="https://www.w3.org/WAI/standards-guidelines/wcag/">Web Content Accessibility Guidelines</a>
-				(WCAG).</p>
+					href="https://www.w3.org/wiki/Open_Web_Platform">Open Web Platform</a>: HTML, CSS, XML, MathML and
+				SVG are the core technologies used for content creation. This means that EPUB
+				publications can be created with a high degree of accessibility simply through the proper application of
+				established digital accessibility techniques.</p>
 
-			<p>In order to outline how to make fully accessible EPUBs, the <a href="https://www.w3.org/TR/epub-a11y-11/"
-					>EPUB Accessibility</a> guidelines have been drafted to address two key needs in the EPUB
-				ecosystem:</p>
+			<p>The <a href="https://www.w3.org/TR/epub-a11y-11/">EPUB Accessibility</a> specifications [epub-a11y-11] address the following key areas for making fully accessible EPUB publications:</p>
 
 			<ul>
 				<li>discoverability of accessibility features of publications;</li>
-				<li>evaluation and certification of accessible EPUB publications.</li>
+				<li>conformance to Web Content Accessibility Guidelines;</li>
+				<li>specific EPUB requirements for accessibility;</li>
+				<li>evaluation and conformance reporting of accessible EPUB publications.</li>
 			</ul>
-
-			<p>Providing accessibility metadata facilitates informed decisions about the usability of an EPUB
-				publication. Consumers can examine the characteristics of the content and decide whether an EPUB
-				publication is appropriate for each need.</p>
-
-			<p>While it has always been possible to create EPUB publications with a high degree of accessibility, this
-				document also establishes formal requirements for contents to be certified as accessible. These
-				requirements provide EPUB creators with a clear set of guidelines for evaluating their content with
-				respect to quality certification and enable it.</p>
-
+			
+			<p>The EPUB Accessibility defines requirements for accessibility metadata. In this way consumers can examine the accessibility characteristics of the content and make well-informed decisions on whether an EPUB publication is appropriate for each need before buying, borrowing or downloading it.
+				This metadata, attached to the digital file and distributed along the supply chain, ensures that persons with disabilities can be informed about the publication accessibility features.
```

Comment:
```suggestion
				This metadata, attached to the digital file and distributed along the supply chain, ensures that persons with disabilities can be informed about the publication accessibility features.
```

Isn't this sentence superfluous? It seems to repeat what is said in the previous sentence.